package.json

```diff
@@ -3,16 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
+    "enzyme": "^2.7.1",
     "eslint": "3.13.1",
     "eslint-plugin-flowtype": "2.29.2",
     "eslint-plugin-react": "6.9.0",
-    "react-scripts": "0.8.4"
+    "react-addons-test-utils": "^15.4.2",
+    "react-scripts": "0.8.4",
+    "react-test-renderer": "*"
   },
   "dependencies": {
     "body-parser": "^1.15.2",
     "node-token-auth": "^1.0.0",
     "react": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react-dom": "^15.4.2",
+    "socket.io-client": "^1.7.3"
```

Comment:
socket.io-client allows the test to see io function as defined, since the test does not have access to <script> tag.